app/helpers/issues/socialization_helper.rb

```diff
@@ -0,0 +1,16 @@
+# Encoding: utf-8
+module Issues
+  module SocializationHelper
+    def toogle_follow_link(issue, size = 'lg')
+      if user_signed_in? && !current_user.follows?(issue)
```

Comment:
You could remake this IF statement.

```
return unless user_signed_in? 

if current_user.follows?(issue)
  // code here
else
 // code here
end
```

It will be more readable and easy to understend